sympy/vector/coordsysrect.py

```diff
@@ -196,6 +250,46 @@ def _set_lame_coefficient_mapping(self, curv_coord_name):
             raise ValueError('Wrong set of parameters. Type of coordinate system is defined')
         self._h1, self._h2, self._h3 = coefficient_mapping[curv_coord_name]
 
+    def _transformation_equations(self, equations):
+        """
+        Helper method for set_coordinate_type. It substitute scalar into symbols.
+
+        Parameters
+        ==========
+
+        equations : tuple
+            Tuple of transformation equations
+
+        """
+        from sympy import symbols
+        x, y, z = symbols('x y z')
```

Comment:
We need something better than this. In `diffgeom` they define the array of variables and the array of equations. Like `arg1=(r, theta, phi), arg2=(eq1, eq2, eq3)`.

Otherwise, support lambdas?